CITATION.cff

```diff
@@ -72,7 +72,47 @@ keywords:
   - "fuzzy matching"
   - "fuzzy search"
 license: "Apache-2.0"
-message: "If you use this software, please cite it using these metadata."
+message: "If you use this software, please cite the following paper."
```

Comment:
```suggestion
message: "If you use this software, please cite the referenced paper."
```
since these data are basically a ``dict``, the keys may not always be in the order like they are in the file.

Reply:
True, thanks.